acceptance/gem_versions/.kitchen.yml

```diff
@@ -0,0 +1,52 @@
+driver:
+  name: ec2
+  instance_type: m3.medium
+  region: us-west-2
+  availability_zone: us-west-2a
+  subnet_id: subnet-19ac017c
+  aws_ssh_key_id: <%= ENV["AWS_SSH_KEY_ID"] %>
+  tags:
+    chef-test: gem_versions
+
+provisioner:
+  name: chef_zero
+
+transport:
+  ssh_key: <%= ENV["HOME"] %>/.ssh/<%= ENV["AWS_SSH_KEY_ID"] %>.pem
```

Comment:
Yeah eventually I would like to have a single set of kitchen.yml files. I would like to target that as the next item.